src/overdue.sh

```diff
@@ -1,20 +1,20 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
-if [ "$EUID" != "0" ] ; then
+if [[ "$EUID" != "0" ]] ; then
     echo 'You must be root to run this program.'
     exit 1
 fi
 
-services="$(lsof -d DEL -F pn0 |\
-    awk -f /usr/share/overdue/overdue.awk |\
-    xargs -L1 systemctl status --no-pager 2>/dev/null |\
-    grep -E '[.]\w+ - ' |\
-    sort | uniq | sort)"
+pids="$(lsof -d DEL | awk 'NR>1 {printf $2" "}')"
```

Comment:
One thing I think we should filter out is deleted handles to `/dev/zero`, that was the original reason the awk script filtered on `$2 ~ /lib\//`, we could probably do something similar here. I've got a couple daemons that sometimes hold open handles to `/dev/zero` and it's always benign.